generic/tests/cfg/lvm.cfg

```diff
@@ -17,6 +17,7 @@
     post_command_noncritical = no
     clean = no
     check_mount = "mountpoint /mnt/kvm_test_lvm"
+    skip_cluster_leak_warn = yes
```

Comment:
Should only apply for `lvm_ioquit`, right?

Reply:
No, lvm_clean also needed.